src/scripts/Build-Toolkit.ps1

```diff
@@ -85,11 +85,17 @@ Get-ChildItem "$PSScriptRoot/../workbooks/$($Template -replace '-workbook$','')*
 | ForEach-Object { Build-QuickstartTemplate $_ }
 
 # Package templates
-Get-ChildItem "$PSScriptRoot/../templates/$Template*" -Directory -ErrorAction SilentlyContinue `
+Get-ChildItem -Path "$PSScriptRoot/../templates/*", "$PSScriptRoot/../optimization-engine*" -Directory -ErrorAction SilentlyContinue `
 | ForEach-Object {
     $srcDir = $_
     $templateName = $srcDir.Name
 
+    # Skip if not the specified template
+    if ($Template -ne "*" -and $Template -ne $templateName)
```

Comment:
FYI - This is just filtering out templates that weren't selected for the build. This cannot be inline anymore since we are pulling from multiple parent directories.